examples/emulator_examples/10_state_example_2_unblock.py

```diff
@@ -0,0 +1,59 @@
+from pybpodapi.protocol import Bpod, StateMachine
+my_bpod = Bpod(emulator_mode=True)
+sma = StateMachine(my_bpod)
+# Set global timer 1 for 3 seconds, following a 1.5 second onset delay after
+# trigger. Link to LED of port 2.
+sma.set_global_timer(timer_id=1, timer_duration=0, on_set_delay=1.5,
+                     channel=Bpod.OutputChannels.PWM2, on_message=255)
+sma.add_state(
```

Comment:
> States 1 - > 8 can also be simplified here.
Right.

> If these tests will be part of a CI (which we need pybpod-api developers on), then it's probably better to decrease the timer duration and the number of states, otherwise we are unnecessarily increasing each test time (and CI bots might be configured to kill and refuse tests that take too long).

If these tests will be part of CI, that is true, and we 'll also have a problem with the threaded emulator examples, where I showcase the manual override functionality.

**But**, these tests were intended to demonstrate emulator functionality in the spirit of the examples under the function_examples and state_machine_examples directories. All these are not unit tests.

I think that CI should not be configured to run these behavioral tests, but only unit tests, if they are added in the future, which I totally recommend.